dev-docs/bidders/flipp.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: bidder
+title: flipp
+description: Prebid Flipp Bidder Adapter
+biddercode: flipp
+media_types: banner
+safeframes_ok: true
+pbjs: true
```

Comment:
This should be `false` as this is a prebid-server only adapter, right?

Reply:
fixed